tests/e2e_oss/README.md

```diff
@@ -0,0 +1,65 @@
+# E2E OSS tests
+
+This folder contains E2E test framework code, E2E tests and related materials
+(reference outputs, test data, etc.)
+
+## Basic scripts
+
+There is one basic script which is responsible for test run with default pipeline - test_base.py. This script perform the
+following actions:
+1. Loads model from its source 
+2. Collects reference from framework
+3. Converts original model through OVC convert model
+4. Performs inference on converted model
+5. Compares reference and output from converted model through element-wise comparison
+
+
+> test_base.py script is run with the help of [pytest](https://docs.pytest.org).
+> The command-line is the same for all the scripts, though some of them
+> __might not__ use specific options.
```

Comment:
Not clear what are basic scripts. Imagine that some external contributor reads this documentation

Reply:
Please refer to my comment above